python/ray/parameter.py

```diff
@@ -46,12 +44,12 @@ class RayParams(object):
             to use, or None for no limit. Once the limit is exceeded, redis
             will start LRU eviction of entries. This only applies to the
             sharded redis tables (task and object tables).
-        object_manager_ports (list): A list of the ports to use for the object
-            managers. There should be one per object manager being started on
-            this node (typically just one).
-        node_manager_ports (list): A list of the ports to use for the node
-            managers. There should be one per node manager being started on
-            this node (typically just one).
+        object_manager_port (int): The port to use for the object managers.
+            There should be one per object manager being started on this node
+            (typically just one).
+        node_manager_port (int): The port to use for the node managers.
+            There should be one per node manager being started on this node
+            (typically just one).
```

Comment:
Here are the changes to parameters. `num_local_schedulers` is removed; `object_manager_ports` and `node_manager_ports` are replaced by single numbers.